python/private/py_wheel.bzl

```diff
@@ -207,12 +207,23 @@ def _input_file_to_arg(input_file):
     return "%s;%s" % (py_package_lib.path_inside_wheel(input_file), input_file.path)
 
 def _py_wheel_impl(ctx):
-    version = _replace_make_variables(ctx.attr.version, ctx)
+    # Make Variable Substitutions
+    variable_index = {}
+    variable_index.update(ctx.var)
+    for dep in ctx.attr.deps:
```

Comment:
I don't think you need special logic to mixin the TemplateVariableInfo values. Just pass the `toolchains` arg. See the "toolchains" attribute here: https://bazel.build/reference/be/common-definitions#common-attributes

Reply:
Aye, I tried pretty hard to do that as that's what [Make Variables](https://bazel.build/reference/be/make-variables) and suggests should work, but just came away with a sore head. If you just try passing in a simple rule like `make_variable_tags` in the examples here to the toolchains argument, then it complains vociferously about it not being a `ToolchainType`.

Would be happy to find out I'm doing something unintelligible though.

Reply:
Ach, just tested it on this simple example and it works. I'll go back and test it on the more complex examples we have in-house, see if I can bissect where it goes wrong.

Reply:
Looks like it's working as a toolchain arg for this simple PR though. Let me go back and see if I can reproduce / bissect the problem on our more complex example.

Reply:
Unable to reproduce. Updated the PR to use and exemplify from passing in a custom rule via the toolchains argument.